Lecture3/Task1/Voucher.java

```diff
@@ -0,0 +1,41 @@
+abstract class Voucher {
+    private String name;
+    private double price;
+    private String country;
+    private String startDate;
+    private int countDays;
+    private int discount;
+    private double cost;
+
+    int getCountDays() {
+        return countDays;
+    }
+
+    double getCost() {
+        return cost;
+    }
+
+    public abstract String getFood();
+
+    public abstract String getTransfer();
+
+    public void initVoucher(int i) {
+        this.name = "Name" + i;
```

Comment:
Please provide more or less valid data for vouchers. countDays and discount are formed from the same variable. It is not valid approach.

Reply:
fixed